packages/block-library/src/site-logo/edit.js

```diff
@@ -122,7 +154,9 @@ const SiteLogo = ( {
 
 	const img = (
 		<img
-			className="custom-logo"
+			className={ classnames( 'custom-logo', {
+				[ borderProps.className ]: !! borderProps.className,
+			} ) }
```

Comment:
```suggestion
			className={ classnames( 'custom-logo', borderProps.className ) }
```

I think the undefined check is unnecessary.